### DISCUSSION TITLE
trash bin

### DISCUSSION THREAD

**Discussion by ferdinandhodl** [iterate-ch/cyberduck#15809] [Ideas]
**Is your feature request related to a problem? Please describe.**
smb and nfs users connecting through their macOS finder, are used to the behavior when "deleting files", the items are moved to the trash bin, e.g. .Trash or #recycle on the remote server.

**Describe the solution you'd like**
Would be helpful to have a setting where the "deleting" behaviour could be specified into "remove on remote server" and "move to trash".

**Describe alternatives you've considered**
using another protocol, such as smb, nfs or afp

**Comment by dkocher** [comment]
We have a feature in the works that will save previous versions instead of deleting on the server in #15137.

**Reply by ferdinandhodl** [reply]
versioning and deleting are fundamentally different concepts from a UI/UX perspective
